packages/gas-fee-controller/src/fetchBlockFeeHistory.test.ts

```diff
@@ -1,18 +1,20 @@
 import { BN } from 'ethereumjs-util';
-import { mocked } from 'ts-jest/utils';
 import { when } from 'jest-when';
-import { query, fromHex, toHex } from '../util';
+import { query, fromHex, toHex } from '@metamask/controller-utils';
```

Comment:
Out of scope for this PR, but `hex` from/to `number/bigint/bytes` is implemented in `@metamask/utils`.